scripts/make/copy-docs.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+import os
+import shutil
+import argparse
+
+
+def is_dir(d):
+    if os.path.isdir(d) and os.access(d, os.R_OK):
+        return d
+    raise Exception("Invalid directory")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-t', '--target_dir', type=is_dir, default='docs/copied-from-beats')
+    parser.add_argument('-s', '--source_dir', type=is_dir, default='../beats/libbeat')
```

Comment:
I don't think we have these files in the repo, do you think this script will affect anything?

Reply:
can be removed